server/server_response.go

```diff
@@ -17,3 +17,14 @@ func sendError(client *Client, headers map[string]string, msg string) {
 	}
 	client.sendChan <- frm.ToUTF8()
 }
+
+func sendMessage(client *Client, headers map[string]string, body []byte) {
+	cmd := frame.MESSAGE
+
+	frm := frame.Frame{
+		Command: cmd,
+		Headers: headers,
+		Body:    frame.Body(body),
+	}
+	client.sendChan <- frm.ToUTF8()
```

Comment:
DESIGN;
It's better to delegate encoding to the client, may be the server could support many types of encoding and it depends on the client.